Documentation/configuration.md

```diff
@@ -34,7 +34,7 @@ The value of the config is a JSON dictionary with the following keys:
    Defaults to the last subnet of `Ipv6Network`.
 
 * `Backend` (dictionary): Type of backend to use and specific configurations for that backend.
-   The list of available backends and the keys that can be put into the this dictionary are listed below.
+   The list of available backends and the keys that can be put into the this dictionary are listed in [Backends](backends.md).
    Defaults to `udp` backend.
```

Comment:
Thanks for contributing.
Could you also fix the default backend that should be vxlan?

Reply:
Yes no prob, sorry for the delay.